### PR TITLE
Excercise 4

### PR DESCRIPTION
**Pull request by slipstreamamx**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: bears-ansible-playbook.YML [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            name: python3
+            state: present
+    become: yes
+
+  - name: install poetry
+    ansible.builtin.shell:
+            cmd: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3
+            creates: /root/.poetry/bin/poetry
+
+    become: yes
```
Comment (BenRamchandani) [inline bears-ansible-playbook.YML:35]: You've overused `become` a bit - if you don't use it here, then poetry will install into `/home/ec2-user` and you can run quite a few of the subsequent commands without root

**Review by BenRamchandani — APPROVED** [review]
This looks good, and I can see the todo app running :+1:

I've left one optional comment about using `become: yes` a bit more than is necessary, but feel free to merge when you're ready.

We'll tear down your VMs next week, so make sure you've got everything you want off of them. If you would like to keep them longer to have a go at the stretch goals or anything let me know.

**Comment by slipstreamamx** [comment]
H Ben,

I would like to keep the environment for another two weeks if possible.

thanks
Asad

**Comment by BenRamchandani** [comment]
> I would like to keep the environment for another two weeks if possible.

Hi Asad, unfortunately I think I tore them down a few months ago. I'm happy to spin a couple of new ones up though if you'd like?

**Pull request merged**